enumerable.rb

```diff
@@ -85,12 +86,13 @@ def my_count(obj = nil)
     counter
   end
 
-  def my_map(_array = nil)
+  def my_map(_obj = nil)
```

Comment:
- **_`#my_map` executes only the proc when both a block and a proc are given and returns a new array with the results of running the proc once for every element in enum._**